.idea/vcs.xml

```diff
@@ -10,5 +10,6 @@
   <component name="VcsDirectoryMappings">
     <mapping directory="" vcs="Git" />
     <mapping directory="$PROJECT_DIR$/backend/ehrenamtskarte-maplibre-style" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$/backend/ehrenamtskarte-maplibre-style/font-glyphs" vcs="Git" />
```

Comment:
You should add to the docs then that this submodule also needs to be initialized (as I did not do this yet ^.^)
Running `git submodule update --init --recursive` in the root repo `entitlementcard` will recursively initialize all submodules.

Reply:
ok so is this even needed then? Should be cloned auto?
```
8. Clone the submodule inside `backend/ehrenamtskarte-maplibre-style`
```

Reply:
you still didnt't tell if you checked if step 8 (clone submodule) is needed? I would like to update the docs

Reply:
It seems I had to update git, so that the submodules would be properly initialized on clone.